public_html/wp-content/themes/wporg-events-2023/src/event-list/index.php

```diff
@@ -47,7 +47,7 @@ function render( $attributes, $content, $block ) {
 	array_walk( $facets, 'sanitize_text_field' );
 	$facets = array_filter( $facets );
 
-	$events = get_events( $attributes['events'], 0, 0, $facets );
+	$events = get_all_upcoming_events();
```

Comment:
@StevenDufresne Did you mean to overwrite this? I'm going to assume not and change it back, since passing `$facets` is needed for filtering, and `get_events()` adds caching, but LMK if there's a reason to use `get_all_upcoming_events()` directly.

Reply:
0889f6f3

Reply:
Eeeek. No, it was to temporarily restore the table for styling purposes.